packages/preview/i-figured/0.1.0/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "i-figured"
+version = "0.1.0"
+authors = ["RubixDev"]
+entrypoint = "i-figured.typ"
+keywords = ["figure"]
+license = "MIT"
+repository = "https://github.com/RubixDev/typst-i-figured"
+description = "Configurable figure numbering per section."
```

Comment:
same as on the other PR, could you exclude the example files with the `exclude` key?

Reply:
done (sorry, here it's a force push instead of a simple commit)